http_parser.c

```diff
@@ -1496,28 +1496,20 @@ size_t http_parser_execute (http_parser *parser,
 
           switch (h_state) {
             case h_general:
-            {
-              const char* p_cr;
-              const char* p_lf;
-              size_t limit = data + len - p;
-
-              limit = MIN(limit, max_header_size);
-
-              p_cr = (const char*) memchr(p, CR, limit);
-              p_lf = (const char*) memchr(p, LF, limit);
-              if (p_cr != NULL) {
-                if (p_lf != NULL && p_cr >= p_lf)
-                  p = p_lf;
-                else
-                  p = p_cr;
-              } else if (UNLIKELY(p_lf != NULL)) {
-                p = p_lf;
-              } else {
-                p = data + len;
+              for (; p != data + len; p++) {
+                ch = *p;
+                if (ch == CR || ch == LF) {
+                  --p;
```

Comment:
If you remove this, can you then skip the `if` below?

Reply:
This is just a revert of the change. I'd rather keep it closer to the original code, unless there are other reasons for improving it here?

Reply:
Sure, that's fine.